### PR TITLE
Added walet unlocking function

### PR DESCRIPTION
**Pull request by elichai**
New function to unlock the wallet from the code.
Should be used with `getpass` library

### CODE REVIEW

File: bitcoin/rpc.py [hunk 1]
```diff
@@ -654,6 +654,18 @@ def validateaddress(self, address):
             r['pubkey'] = unhexlify(r['pubkey'])
         return r
 
+    def unlockwallet(self, password, timeout=60):
+    	"""Stores the wallet decryption key in memory for 'timeout' seconds.
+    	Should be used with `getpass` library
```
Comment (petertodd) [inline bitcoin/rpc.py]: Why should this be used with `getpass` specifically?

**Comment by elichai** [comment]
So it won't echo the password into the terminal.

File: bitcoin/rpc.py (hunk 1 above)
Comment (elichai) [inline bitcoin/rpc.py]: So it won't echo the password into the terminal.

Comment (elichai) [inline bitcoin/rpc.py]: I'll reopen a new pull request

**Comment by elichai** [comment]
Opened new pull request

**Pull request closed without merging**